### DISCUSSION TITLE
[FEATURE]: Franchise history and individual user history for all time stats

### DISCUSSION THREAD

**Discussion by brian-dev8** [joeyagreco/leeger#36] [Ideas]
### Notes

Currently leeger appears to grab only franchise history for a league, so each team year after year is basically expected to be the same user or relatively the same team (if dynasty for example).

Also, would be nice if there was a flag or switch to break it down by user, so as a old owner is replaced, you could see the new owner history from the time they joined the league for all owners.  So you would end up with more records than teams since you could have many new owners over the history of a league.  

For example, yahoo has a specific user guid, so you could organize each user by guid for every year to group them together.

**Comment by joeyagreco** [comment]
This could definitely be a useful feature, especially for leagues that have been around for a while.

In your mind, how would this be displayed for the user in Excel?

Would the "All Time" tab just have each owner that has ever been a part of the league but all of the Year tabs remain the same?

**Comment by brian-dev8** [comment]
yes exactly what I was thinking
